src/main/java/org/akaza/openclinica/controller/StudyParticipantController.java

```diff
@@ -610,13 +610,25 @@ private String startBulkCaseBookPDFJob(String schema,
 										   String landscape) {
 									 	 
 
+		    String siteName = null;
+		    String studyName = null;
+		    String pdfHeader =  "StudyName: SiteName - Participant ParticipantID";
```

Comment:
Instead of using string replacement, use MessageFormat https://docs.oracle.com/javase/8/docs/api/java/text/MessageFormat.html

Reply:
done